src/routes/Profile/SomethingElse/EditMuuOsaaminenModal.tsx

```diff
@@ -19,7 +19,7 @@ interface OsaamisetForm {
   osaamiset: OsaaminenValue[];
 }
 
-const EditMuuOsaaminenModal = ({ isOpen, onClose }: EditMuuOsaaminenModal) => {
+const EditMuuOsaaminenModal = ({ isOpen, onClose }: EditMuuOsaaminenModalProps) => {
```

Comment:
Here also, the interface and the component had the same name and it was conflicting.